Make `core.get_node_raw` a public API (#15332)
minetest/minetest: doc/lua_api.md

* `core.get_node_or_nil(pos)`
    * Same as `get_node` but returns `nil` for unloaded areas.
    * Note that even loaded areas can contain "ignore" nodes.

> * `core.get_node_raw(x, y, z)`
>     * Low-level API returning numbers, not tables
>     * Returns `content_id`, `param1`, `param2`, and `pos_ok`
>     * The `content_id` can be mapped to a node name via `core.get_name_from_content_id`
>     * If `pos_ok` is false, the node is not yet loaded and the `content_id` is that of "ignore" nodes

Reviewer: ```suggestion
* `core.get_node_raw(x, y, z)`
    * Faster low-level API that returns numbers, not a table
    * Returns `content_id`, `param1`, `param2`, and `pos_ok`
    * The `content_id` can be mapped to a name using `core.get_name_from_content_id()`
    * If `pos_ok` is false, the node is not loaded and `content_id == core.CONTENT_IGNORE`
```
